src/stlink-lib/chipid.c

```diff
@@ -1121,11 +1122,45 @@ void init_chipids(char *dir_to_scan)
       dump_a_chip (stderr, op);
       fprintf (stderr, "---------- new ------------\n");
       dump_a_chip (stderr, p);
-      
+
```

Comment:
```suggestion
      
```

Reply:
I'd prefer to avoid white space where not needed for improved readability.

Reply:
I agree, but I don’t like mixing coding style changes with other ones. So, if my suggestion was accepted, this line would not be changed at all.

Reply:
Should be ok for now. We might unify this during clean-up proceedings at a later point in time.